pkg/proxy/server_test.go

```diff
@@ -486,6 +488,7 @@ func TestServerHTTP_Secure_DelayTx(t *testing.T)   { testServerHTTP(t, true, tru
 func TestServerHTTP_Insecure_DelayRx(t *testing.T) { testServerHTTP(t, false, false) }
 func TestServerHTTP_Secure_DelayRx(t *testing.T)   { testServerHTTP(t, true, false) }
 func testServerHTTP(t *testing.T, secure, delayTx bool) {
+	t.Helper()
```

Comment:
Don't agree with that change. From what I know calling `t.Helper()` will change which line failure is attributed? https://pkg.go.dev/testing#T.Helper.

I'm pretty sure this function is not a helper.

This kind of flake make me think we should skip this linter.